modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -612,12 +620,22 @@ private CompletableFuture<Object> finishTransaction(List<ReplicationGroupId> agg
         HybridTimestamp commitTimestamp = hybridClock.now();
 
         CompletableFuture<Object> changeStateFuture = raftClient.run(
-                new FinishTxCommand(
-                        txId,
-                        commit,
-                        commitTimestamp,
-                        aggregatedGroupIds
-                )
+                msgFactory.finishTxCommand()
+                        .txId(txId)
+                        .commit(commit)
+                        .commitTimestamp(
+                                msgFactory.hybridTimestampMessage()
```

Comment:
I think you have a `hybridTimestamp(...)` method in this class that'll make this code shorter

Reply:
Done.